pybossa/cache/projects.py

```diff
@@ -206,7 +205,7 @@ def get_all_featured(category=None):
     sql = text('''SELECT project.id, project.name, project.short_name, project.info,
                project.created, project.updated, project.description,
                "user".fullname AS owner FROM project, "user"
-               WHERE project.featured=true AND project.hidden=0
+               WHERE project.featured=true
```

Comment:
It could be interesting to have a constraint on the DB: http://www.postgresql.org/docs/9.3/static/ddl-constraints.html